components/description-list/description-list-wrapper.js

```diff
@@ -86,6 +90,18 @@ class DescriptionListWrapper extends LitElement {
 		return html`<slot class="${classMap(classes)}"></slot>`;
 	}
 
+	updated(changedProperties) {
+		if (changedProperties.get('forceStacked')) {
+			if (!this.forceStacked) {
+				this._resizeObserver = new ResizeObserver(this._onResize.bind(this));
+				this._resizeObserver.observe(this);
+			} else {
+				this._resizeObserver = undefined;
+				this._stacked = true;
+			}
+		}
+	}
```

Comment:
You're going to want to disconnect it when it changes to `true` as well (if it was connected in the first place).

```suggestion
	updated(changedProperties) {
		if (changedProperties.has('forceStacked')) {
			if (!this.forceStacked) {
				this._resizeObserver = new ResizeObserver(this._onResize.bind(this));
				this._resizeObserver.observe(this);
			} else {
				if (this._resizeObserver) {
					this._resizeObserver.disconnect();
					this._resizeObserver = undefined;
				}
				this._stacked = true;
			}
		}
	}
```

Reply:
agh yes 🤦